packages/drawer/src/Drawer.stories.tsx

```diff
@@ -117,9 +119,8 @@ const TemplateComponent: StoryFn<DrawerProps> = ({
   const renderDrawer = () => (
     <Drawer
       {...rest}
-      displayMode={displayMode}
-      open={open}
       onClose={() => setOpen(false)}
+      open={undefined} // Prevent open from passing since it is passed to DrawerLayout
```

Comment:
do we need to do this explicitly, or can we just leave it unset? AFAI can see it's not being spread into the component by `...rest`, so we don't need to override it